### PR TITLE
Rebalance Johto & Hoenn gyms, fix Tate & Liza

### PR DESCRIPTION
**Pull request by apple096**
Health too low

### CODE REVIEW

File: src/scripts/dungeons/Dungeon.ts [hunk 1]
```diff
@@ -145,95 +145,95 @@ dungeonList['Sprout Tower'] = new Dungeon('Sprout Tower',
...
     3500, BadgeCase.Badge.Zephyr, 33, 12
 );
 
 dungeonList['Ilex Forest'] = new Dungeon('Ilex Forest',
     ['Caterpie', 'Metapod', 'Weedle', 'Kakuna', 'Zubat', 'Oddish', 'Paras', 'Hoothoot'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.xClick, GameConstants.BattleItemType.Token_collector],
-    1200,
-    [new DungeonBossPokemon('Noctowl', 5000, 30), new DungeonBossPokemon('Beedrill', 5000, 30), new DungeonBossPokemon('Butterfree', 5000, 30), new DungeonBossPokemon('Celebi', 300000, 50)],
+    41200,
+    [new DungeonBossPokemon('Noctowl', 150000, 30), new DungeonBossPokemon('Beedrill', 5000, 30), new DungeonBossPokemon('Butterfree', 5000, 30), new DungeonBossPokemon('Celebi', 300000, 50)],
```
Comment (RedSparr0w) [inline src/scripts/dungeons/Dungeon.ts]: The other boss Pokemon in this dungeon also need to be modified.

_Same thing for the other dungeons that have been changed_

Comment (apple096) [reply src/scripts/dungeons/Dungeon.ts]: Whoops

Comment (apple096) [reply src/scripts/dungeons/Dungeon.ts]: Fixed

**Comment by apple096** [comment]
I think I fixed all of them

File: src/scripts/dungeons/Dungeon.ts [hunk 2]
```diff
@@ -242,122 +242,122 @@ dungeonList['Mt Silver'] = new Dungeon('Mt Silver',
...
+    [new DungeonBossPokemon('Whismur', 250000, 20)],
     12000, BadgeCase.Badge.Stone, 101, 5);
 
 dungeonList['Granite Cave'] = new Dungeon('Granite Cave',
     ['Zubat', 'Abra', 'Geodude', 'Makuhita', 'Aron', 'Sableye'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xExp],
-    4000,
-    [new DungeonBossPokemon('Mawile', 100000, 20), new DungeonBossPokemon('Nosepass', 100000, 20)],
+    101000,
+    [new DungeonBossPokemon('Mawile', 260000, 20), new DungeonBossPokemon('Nosepass', 100000, 20)],
```
Comment (RedSparr0w) [inline src/scripts/dungeons/Dungeon.ts]:
```suggestion
    [new DungeonBossPokemon('Mawile', 260000, 20), new DungeonBossPokemon('260000', 100000, 20)],
```

File: src/scripts/dungeons/Dungeon.ts [hunk 3]
```diff
@@ -242,122 +242,122 @@ dungeonList['Mt Silver'] = new Dungeon('Mt Silver',
...
+    [new DungeonBossPokemon('Whismur', 250000, 20)],
     12000, BadgeCase.Badge.Stone, 101, 5);
 
 dungeonList['Granite Cave'] = new Dungeon('Granite Cave',
     ['Zubat', 'Abra', 'Geodude', 'Makuhita', 'Aron', 'Sableye'],
     [GameConstants.BattleItemType.xAttack, GameConstants.BattleItemType.xExp],
-    4000,
-    [new DungeonBossPokemon('Mawile', 100000, 20), new DungeonBossPokemon('Nosepass', 100000, 20)],
+    101000,
+    [new DungeonBossPokemon('Mawile', 260000, 20), new DungeonBossPokemon('260000', 100000, 20)],
```
Comment (RedSparr0w) [inline src/scripts/dungeons/Dungeon.ts]:
```suggestion
    [new DungeonBossPokemon('Mawile', 260000, 20), new DungeonBossPokemon('Nosepass', 260000, 20)],
```

**Review by RedSparr0w — APPROVED** [review]
Nice work, Thanks for the PR!

File: src/scripts/dungeons/Dungeon.ts (hunk 3 above)
Comment (apple096) [reply src/scripts/dungeons/Dungeon.ts]: Oops
Sorry

**Pull request merged**